dao.py

```diff
@@ -0,0 +1,49 @@
+from flask_pymongo import PyMongo
+from bson.objectid import ObjectId
+
+from app import app
+
+DB_NAME = 'schedule_practice'
+DB_USERNAME = 'admin'
+DB_PASSWORD = 'admin'
+DB_URI = 'mongodb://%s:%s@ds239387.mlab.com:39387/schedule_practice' % (DB_USERNAME, DB_PASSWORD)
+
+class DAO:
+    app.config['MONGO_DBNAME'] = DB_NAME
+    app.config['MONGO_URI'] = DB_URI
+    __mongo = PyMongo(app)
```

Comment:
Instead of utilizing the entire `PyMongo` library .. look into the `MongoClient`
```py
from pymongo import MongoClient
```

Reply:
So is `flask-pymongo` not necessary for this project?